### PR TITLE
Add instructions for the UMD build to the `Quick start` guide

### PR DESCRIPTION
**Pull request by filipsobol**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: Add instructions for the UMD build to the `Quick start` guide.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

**Comment by Witoso** [comment]
Please sync this with what @gorzelinski started to prepare in the cloud branch.

**Review by godai78 — DISMISSED** [review]
LGTM. We must keep in mind Melman sync for this, too.

File: docs/getting-started/quick-start.md [hunk 1]
```diff
@@ -193,6 +200,108 @@ Your final page should look similar to the one below.
...
+
+	ClassicEditor
+		.create( document.querySelector( '#editor' ), {
+			plugins: [ Essentials, Bold, Italic, Font, Paragraph ],
+			toolbar: {
+				items: [
+					'undo', 'redo', '|', 'bold', 'italic', '|',
+					'fontSize', 'fontFamily', 'fontColor', 'fontBackgroundColor'
+				]
+			}
```
Comment (gorzelinski) [inline docs/getting-started/quick-start.md:242]: I'm still not sure which version we prefer. My thought process is - if we don't use additional settings, let's use the simplified configuration without nested objects. But maybe there are counterarguments.

```suggestion
			toolbar: [
					'undo', 'redo', '|', 'bold', 'italic', '|',
					'fontSize', 'fontFamily', 'fontColor', 'fontBackgroundColor'
				]
```

File: docs/getting-started/quick-start.md [hunk 2]
```diff
@@ -269,16 +378,18 @@ That is all the code you need to see a bare-bone editor running in your web brow
 
 ### Installing premium features from CDN
 
-Just like with open-source features, start by attaching a link to style sheets. They contain all styles for the editor's UI and content. The styles are in two separate style sheets &ndash; for open-source and premium plugins. You can also include your styles if you like. Refer to the content styles guide for more information.
+Just like with open-source features, you can choose between using ES Modules build with imports or UMD build with global variables. However, these methods should not be mixed. You should use the same method for both the open-source and premium features.
```
Comment (gorzelinski) [inline docs/getting-started/quick-start.md:381]:
```suggestion
Just like with open-source features, you can choose between using an ES Modules build with imports or a UMD build with global variables. However, these methods should not be mixed. You should use the same method for both the open-source and premium features.
```

**Review by gorzelinski — COMMENTED** [review]
I added a few comments.

File: docs/getting-started/quick-start.md [hunk 3]
```diff
@@ -144,6 +148,9 @@ Lastly, add a tag for the editor to attach to.
 
 Your final page should look similar to the one below.
 
+<details>
+<summary>Final build</summary>
```
Comment (Witoso) [inline docs/getting-started/quick-start.md:152]: I don't like `build` world :grin:

**Comment by Witoso** [comment]
@gorzelinski is this similar to what you have on your branch? I feel we are touching this from different angles, but I'm not sure.

File: docs/getting-started/quick-start.md [hunk 4]
```diff
@@ -144,6 +148,9 @@ Lastly, add a tag for the editor to attach to.
 
 Your final page should look similar to the one below.
 
+<details>
```
[the remark this answers is not in the document]
Comment (godai78) [reply docs/getting-started/quick-start.md:151]: I'd say no, even if we have one very loing. We hardly ever did it in the past, I believe, only started with examples that had large listings.

File: docs/getting-started/quick-start.md (hunk 1 above)
Comment (godai78) [reply docs/getting-started/quick-start.md:242]: I think we mostly used `items` if we had vertical listings, but there was no fixed rule. I'm for this change to make it simplr.

**Comment by gorzelinski** [comment]
> @gorzelinski is this similar to what you have on your branch? I feel we are touching this from different angles, but I'm not sure.

Here's a table of contents of the vanilla js cloud quick start on the epic branch. There is an "Installing CKEditor 5 from CDN" section where I describe the usage with global variables and UMD modules. The "Installing premium features from CDN" is the same but for premium features.

The "Advanced setup with import maps" section contains the setup with import maps and ES Modules.

So, the sections are similar. However, nowhere in the guide do I mention "build," "UMD," or "ES modules" directly.

Screenshot 2024-09-03 at 11 28 51

File: docs/getting-started/quick-start.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (gorzelinski) [reply docs/getting-started/quick-start.md:151]: > I'd say no, even if we have one very loing. We hardly ever did it in the past, I believe, only started with examples that had large listings.

Yeah, that's my thoughts too.

File: docs/getting-started/quick-start.md (hunk 3 above)
Comment (godai78) [reply docs/getting-started/quick-start.md:152]: "Final solution" is both much better and much worse.

File: docs/getting-started/quick-start.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (filipsobol) [reply docs/getting-started/quick-start.md]: I removed all `<details>` :+1:

**Review by godai78 — DISMISSED** [review]

**Review by godai78 — APPROVED** [review]

File: docs/getting-started/quick-start.md [hunk 5]
```diff
@@ -233,20 +324,18 @@ import 'ckeditor5-premium-features/ckeditor5-premium-features.css';
...
+			'formatPainter'
+		],
 		licenseKey: '<YOUR_LICENSE_KEY>'
 	} )
 	.then( /* ... */ )
 	.catch( /* ... */ );
 ```
 
-Pass the imported plugins inside the configuration to the {@link module:editor-classic/classiceditor~ClassicEditor#create `create()`} method and add toolbar items where applicable. Please note, that to use premium features, you need to activate them with a proper license key, mentioned in the final section of this guide.
+Pass the imported plugins inside the configuration to the {@link module:editor-classic/classiceditor~ClassicEditor#create `create()`} method and add toolbar items where applicable. Please note, that to use premium features, you need to activate them with a proper license key, as described in the [final section of this guide](#obtaining-a-license-key).
```
Comment (gorzelinski) [inline docs/getting-started/quick-start.md:338]:
```suggestion
Pass the imported plugins inside the configuration to the {@link module:editor-classic/classiceditor~ClassicEditor#create `create()`} method and add toolbar items where applicable. Please note that to use premium features, you need to activate them with a proper license key, as described in the [final section of this guide](#obtaining-a-license-key).
```

**Review by gorzelinski — DISMISSED** [review]
LGTM

**Review by godai78 — APPROVED** [review]

**Review by gorzelinski — APPROVED** [review]

**Pull request merged**